Shared/Coordinator/MainCoordinator+Extensions.swift

```diff
@@ -10,28 +10,23 @@ import Foundation
 import ParseLiveQuery
 
 extension MainCoordinator: UserNotificationManagerDelegate {
-    func userNotificationManager(willHandle deeplink: DeepLinkable) {
-        self.deepLink = deeplink
-        self.handle(deeplink: deeplink)
+    
+    nonisolated func userNotificationManager(willHandle deeplink: DeepLinkable) {
```

Comment:
Because coordinators are now run on the MainActor by default (basically the main thread), we need to mark particular functions as non-isolated if they might be called off the MainActor.

So in this case, we might get a deep link triggered off of some background task. Within the function we can then call our code back on the MainActor.